### PR TITLE
update to version 1 compatible

### PR DESCRIPTION
**Pull request by calvinmetcalf**

### CODE REVIEW

**Comment by calvinmetcalf** [comment]
also, tests for everything

**Comment by frewsxcv** [comment]
Wow this is great! I'll look it over tonight

File: src/lib.rs [hunk 1]
```diff
@@ -15,162 +15,210 @@
...
 ///
 /// [GeoJSON Format Specification § 2.1.1](http://geojson.org/geojson-spec.html#positions)
+#[derive(RustcEncodable, Clone)]
 pub struct Pos(pub Vec<f64>);
 
 impl ToJson for Pos {
     fn to_json(&self) -> json::Json {
-        let &Pos(ref nums) = self;
-        nums.to_json()
+        match self.clone() {
```
Comment (frewsxcv) [inline src/lib.rs]: Are you able to do `self.clone().to_json()`?

**Comment by frewsxcv** [comment]
There's some formatting issues, but I'm not worried about them. It looks good to me after that comment above gets addressed

File: src/lib.rs (hunk 1 above)
Comment (calvinmetcalf) [inline src/lib.rs]: that leads to a stack overflow, though the ref syntax you originally had does do the trick

**Comment by frewsxcv** [comment]
Awesome, thanks @calvinmetcalf for all of this

**Pull request merged**